app/src/main/java/casino/poker/PokerHand.java

```diff
@@ -78,8 +78,23 @@ public boolean isFlush() { // Unit test
     }
 
     public boolean isStraight() { // Unit test for all of the methods
```

Comment:
Where's the unit tests for this method?

Reply:
Might be nice to add some comments to this code to explain it for future reference.